### ISSUE TITLE
No module named 'png'

### ISSUE THREAD

**Issue by theautomation** [DeerMaximum/QR-Code-Generator#58]
## Version of the custom_component
v.2.0.1

## Describe the bug
Can't generate the png file

Getting an error that 'png' python module is missing. I am using Home Assistant from the offical docker image. I can install the png module with a command in the Dockerfile but don't see any requirements in your readme.

## log
```text
Logger: homeassistant.components.image
Source: custom_components/qr_generator/image.py:126
Integration: Image (documentation, issues)
First occurred: 10:37:42 PM (8 occurrences)
Last logged: 10:41:03 PM

Error adding entities for domain image with platform qr_generator
Error while setting up qr_generator platform for image
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 510, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 750, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1003, in add_to_platform_finish
    await self.async_added_to_hass()
  File "/config/custom_components/qr_generator/image.py", line 94, in async_added_to_hass
    await self._refresh()
  File "/config/custom_components/qr_generator/image.py", line 126, in _refresh
    code.png(
  File "/usr/local/lib/python3.11/site-packages/pyqrcode/__init__.py", line 462, in png
    builder._png(self.code, self.version, file, scale,
  File "/usr/local/lib/python3.11/site-packages/pyqrcode/builder.py", line 1266, in _png
    import png
ModuleNotFoundError: No module named 'png'
```

**Comment by DeerMaximum** [comment]
The requirements are stored in the `manifest.json`. So the pypng packet. 
Home Assistant should install the requirements automatically.


manifest.json
```
...
"requirements": ["pyqrcode==1.2.1", "pypng==0.0.21", "pillow==10.0.0"]
...
```

**Comment by theautomation** [comment]
Well I am using the latest docker image (2023.8.2) and looks like the pypng package is missing. 
```bash
home-assistant-6c56494d85-nmnb5:/config# pip3 show pypng
WARNING: Package(s) not found: pypng
home-assistant-6c56494d85-nmnb5:/config# 
```

When ik manually install the package `pip3 install pypng' in the container the integration work and can see the qrcode.

**Comment by DeerMaximum** [comment]
When you set up the integration for the first time, you should find the following log line in the logs. If not, that's an error from Home Assistant, I can't help you with that.
```
2023-08-13 22:51:21.065 INFO (SyncWorker_1) [homeassistant.util.package] Attempting install of pypng==0.0.21
```

**Comment by Aleksey-nt** [comment]
After being updated on Home AS OC 2023.8, does not work. 

ERROR: Setup failed for custom integration qr_generator: Requirements for qr_generator not found: ['pillow==9.5.0'].

Unable to install package pillow==9.5.0: ERROR: Cannot install pillow==9.5.0 because these package versions have conflicting dependencies. ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts [notice] A new release of pip is available: 23.1.2 -> 23.2.1 [notice] To update, run: pip install --upgrade pip

**Comment by DeerMaximum** [comment]
@Aleksey-nt please see #54
